components/dashboard/src/projects/ConfigureProject.tsx

```diff
@@ -62,14 +62,13 @@ export default function () {
   const [ workspaceCreationResult, setWorkspaceCreationResult ] = useState<WorkspaceCreationResult | undefined>();
 
   useEffect(() => {
-    if (!team) {
```

Comment:
@jankeromnes should we add a check for "`teams` loaded" here as well?

Reply:
Uh... yes. 😳 I thought I got them all. Probably need more ☕!